### ISSUE TITLE
PIP Package

### ISSUE THREAD

**Issue by fakerybakery** [myshell-ai/MeloTTS#24]
Hi,
This is already pip-packaged, but it is not published on PyPI. This means:

1. It cannot be specified as a requirement on other packages published on PyPI
2. It must be installed via Git

Do you think it might be possible to publish it on PyPI in the future? If so, I'd be happy to make a PR that will use GitHub Actions to automatically publish the package on new updates.

Thanks!

**Comment by Zengyi-Qin** [comment]
Sure! Let's publish it on PyPI. Could you help make a PR?

**Comment by fakerybakery** [comment]
Sure! Before the package can be published to PyPI, it must first be published manually. Here are the steps:
1. Register for PyPI [here](https://pypi.org/account/register/)
2. Git clone MeloTTS, cd, etc
3. Needs a unique name, opened #25
4. Build distributable:
```bash
python setup.py sdist
pip install twine
twine upload dist/*
```
5. Go to PyPI > Settings, generate a Write-access API key for the MeloTTS project. Go to Settings in the GitHub repo > Secrets and Variables > Actions > New repository secret. Name it `PYPI_API_TOKEN` and set the value to the API token
6. Whenever there is a major update, bump the version in setup.py and create a new [GitHub release](https://github.com/myshell-ai/MeloTTS/releases)

I'll open a PR, but these steps must first be completed before the PR will work.
Thanks!

**Comment by Zengyi-Qin** [comment]
Thank you! Let me follow these steps

**Comment by Zengyi-Qin** [comment]
All the above step is completed

**Comment by fakerybakery** [comment]
Found a bug, addressed in #28. After it is fixed, MeloTTS can be installed with `pip install melotts`

**Comment by GordonWu** [comment]
@fakerybakery Hi, quick note: got an error when trying to pip install melotts.

![image](https://github.com/myshell-ai/MeloTTS/assets/6059222/945b2cf2-fb19-4129-9973-639f8d616823)

**Comment by fakerybakery** [comment]
Yes, that is resolved in #28
